File: packages/expo-dev-menu/vendored/react-native-safe-area-context/android/devmenu/com/th3rdwave/safeareacontext/Rect.java
package devmenu.com.th3rdwave.safeareacontext;

/* package */ class Rect {
  float x;
  float y;
  float width;
  float height;

  Rect(float x, float y, float width, float height) {
    this.x = x;
    this.y = y;
    this.width = width;
    this.height = height;
  }

  boolean equalsToRect(Rect other) {
    if (this == other) {
      return true;
    }
    return this.x == other.x && this.y == other.y && this.width == other.width && this.height == other.height;
  }
}

File: packages/expo-dev-menu/vendored/react-native-safe-area-context/android/devmenu/com/th3rdwave/safeareacontext/Rect.kt
package devmenu.com.th3rdwave.safeareacontext

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record

data class Rect(
  @Field val x: Float,
  @Field val y: Float,
  @Field val width: Float,
  @Field val height: Float
) : Record
